app/src/main/java/org/coepi/android/ui/extensions/PublicReportExtensions.kt

```diff
@@ -0,0 +1,43 @@
+package org.coepi.android.ui.extensions
+
+import org.coepi.android.R.string
+import org.coepi.android.api.publicreport.CoughSeverity
+import org.coepi.android.api.publicreport.CoughSeverity.DRY
+import org.coepi.android.api.publicreport.CoughSeverity.EXISTING
+import org.coepi.android.api.publicreport.CoughSeverity.WET
+import org.coepi.android.api.publicreport.FeverSeverity
+import org.coepi.android.api.publicreport.FeverSeverity.MILD
+import org.coepi.android.api.publicreport.FeverSeverity.SERIOUS
+import org.coepi.android.api.publicreport.PublicReport
+import org.coepi.android.system.Resources
+
+
+
+fun FeverSeverity.toUIString(resources: Resources): String? =
+    when (this) {
+        FeverSeverity.NONE -> null
+
+        MILD -> resources.getString(string.alerts_fever_report_detail,
+            resources.getString(string.symptom_report_fever_severity_mild))
+
+        SERIOUS -> resources.getString(string.alerts_fever_report_detail,
+                resources.getString(string.symptom_report_fever_severity_serious))
+    }
+
+fun CoughSeverity.toUIString(resources: Resources): String? =
+    when (this) {
+        CoughSeverity.NONE -> null
+        EXISTING -> resources.getString(string.alerts_cough_report)
+
+        WET -> resources.getString(string.alerts_cough_report_detail,
```

Comment:
Minor: You can use static imports to omit `string.` here: Press alt+enter on the string's name and the IDE should offer it. It can be a bit annoying when copy pasting code but it's better for readability.